External/Plugins/ASCompletion/Model/ASFileParser.cs

```diff
@@ -1189,7 +1189,7 @@ public void ParseSrc(FileModel fileModel, string ba, bool allowBaReExtract)
                     else
                     {
                         // valid chars for identifiers
-                        if (c1 >= 'A' && c1 <= 'Z')
+                        if (char.IsLetter(c1))
```

Comment:
I want to modify this part, because it's possible, but in Haxe I think it depends on some factors and so I prefer it to be disabled for that one, unless I made something wrong in my tests, but for AS is perfectly valid (AS allows for some strange things heh, and allows for some strange Unicode characters to be used as the replacement of other things). Also, $ is not valid in Haxe.

Processing is not affected that much with this method, it's already quite optimized, of course there it's slower, but really not that much.